pepper-apis/src/main/java/org/broadinstitute/ddp/model/pdf/PdfConfiguration.java

```diff
@@ -55,11 +55,12 @@ public PdfVersion getVersion() {
         return version;
     }
 
-    public List<PdfTemplate> getTemplates() {
-        return templates;
+    public List<Long> getTemplateIds() {
+        return templateIds;
     }
 
-    public void addTemplate(PdfTemplate template) {
-        templates.add(template);
+    public void addTemplateId(Long templateId) {
```

Comment:
Let's use primitive `long` to avoid possibility of `null`s.
```suggestion
    public void addTemplateId(long templateId) {
```